axon-server-connector/src/main/java/org/axonframework/axonserver/connector/query/subscription/AxonServerSubscriptionQueryResult.java

```diff
@@ -69,7 +75,7 @@ public AxonServerSubscriptionQueryResult(final io.axoniq.axonserver.connector.qu
     public AxonServerSubscriptionQueryResult(final SubscriptionQueryMessage<?, ?, ?> queryMessage,
                                              final io.axoniq.axonserver.connector.query.SubscriptionQueryResult result,
                                              final SubscriptionMessageSerializer subscriptionSerializer,
-                                             final SpanFactory spanFactory,
```

Comment:
Although bothersome, this is a breaking change requiring the deprecation of this constructor and the introducation of another constructor.

Reply:
Agreed! But I also think this class shouldn't have been made public at the start of AF 4.0....sorry to _bother_ you with that bit of legacy.